mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -19,6 +19,8 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
 }
 
 ::Const.TipOfTheDay.extend([
+	"Reforged: When a character is killed by someone in combat, positive morale checks are triggered only for allies of the killer, instead of for all enemies.",
+	"Reforged: Morale checks from dying/fleeing characters are stronger when the dying/fleeing character is more powerful - and vice versa.",
```

Comment:
You could write `that` instead of the second `dying/fleeing`. You decide

Reply:
I like that change. However, we can go even further and do something like: `Morale checks due to relatively more powerful characters dying or fleeing are proportionately stronger, and vice versa.` I think this is concise and reads better.